certbot/compat/os.py

```diff
@@ -29,3 +29,15 @@
 
 # Clean all remaining importables that are not from the core os module.
 del ourselves, std_os, std_sys
+
+
+# The os.open function on Windows will have the same effect than a bare os.chown towards the given
+# mode, and will create a file with the same flaws that what have been described for os.chown.
+# So upon file creation, security.take_ownership will be called to ensure current user is the owner
+# of the file, and security.chmod will do the same thing than for the modified os.chown.
```

Comment:
security.chmod will do what thing?

Reply:
should this be "then", as in "after"?

Reply:
Not relevant anymore, since I rewrote the comment.